revolv/ambassador/views.py

```diff
@@ -2,21 +2,31 @@
 
 from django.views.generic import TemplateView
 from revolv.base.users import UserDataMixin
+from revolv.base.utils import ProjectGroup
 from revolv.project.models import Project
 
 
 class AmbassadorDashboardView(UserDataMixin, TemplateView):
     """Basic view for the Ambassador dashboard.
     """
-    template_name = 'ambassador/dashboard.html'
+    template_name = 'base/dashboard.html'
 
     def get_context_data(self, **kwargs):
+        # context = super(AdministratorDashboardView, self).get_context_data(**kwargs)
         context = super(AmbassadorDashboardView, self).get_context_data(**kwargs)
         user_projects = Project.objects.owned_projects(self.user)
-        context['drafted_projects'] = Project.objects.get_drafted(user_projects)
-        context['proposed_projects'] = Project.objects.get_proposed(user_projects)
-        context['active_projects'] = Project.objects.get_active(user_projects)
-        context['completed_projects'] = Project.objects.get_completed(user_projects)
-        context['all_projects'] = list(chain(context['drafted_projects'], context['proposed_projects'],
-                                             context['active_projects'], context['completed_projects']))
+
+        project_dict = {}
+        project_dict[ProjectGroup('Drafted Projects', "drafted")] = Project.objects.get_drafted(user_projects)
+        project_dict[ProjectGroup('Proposed Projects', "proposed")] = Project.objects.get_proposed(user_projects)
+        project_dict[ProjectGroup('Active Projects', "active")] = Project.objects.get_active(user_projects)
+        project_dict[ProjectGroup('Completed Projects', "completed")] = Project.objects.get_completed(user_projects)
+
+        context["project_dict"] = project_dict
+        context["all_projects"] = list(chain(*(project_dict.values())))
+        context["role"] = "ambassador"
```

Comment:
We do need role, because in the case where the administrator looks at the admin dashboard, we need to know that they're looking as an ambassador rather than an admin.

As for the repetition, good point :) I'm not even actually sure if we need "all_projects" anymore, so I'll at least take that out.